lib/assert.js

```diff
@@ -69,12 +70,12 @@ function getStack() {
 	return obj.stack;
 }
 
-function validateExpectations(assertion, expectations, numArgs) { // eslint-disable-line complexity
+function validateExpectations(assertion, expectations) { // eslint-disable-line complexity
 	if (typeof expectations === 'function') {
 		expectations = {instanceOf: expectations};
 	} else if (typeof expectations === 'string' || expectations instanceof RegExp) {
 		expectations = {message: expectations};
-	} else if (numArgs === 1 || expectations === null) {
+	} else if (expectations === null || expectations === undefined) {
```

Comment:
Checking the argument length was an explicit change to address https://github.com/avajs/ava/issues/1676. We need to retain that behavior.

Reply:
:+1: Sorry, was not aware.